cms/templates/asset_index.html

```diff
@@ -35,32 +35,31 @@
 
 <div class="wrapper-mast wrapper">
     <header class="mast has-actions has-subtitle">
-        <h1 class="page-header">
+        <h3 class="page-header">
             <small class="subtitle">${_("Content")}</small>
-            <span class="sr">&gt; </span>${_("Files & Uploads")}
-        </h1>
+            <span class="sr">- </span>${_("Files & Uploads")}
```

Comment:
@cahrens Originally, it was a greater than symbol, indicating the relationship between the word Content and Files & Uploads, but while the greater than symbol visually indicates that relationship to us, its not so effective when read out by a screen reader (GREATER THAN) so I suggested @sarahkf replace it with a dash, which is a better convention for indicating this relationship.